tao_compiler/mlir/xla/ral/ral_helper.h

```diff
@@ -64,7 +64,10 @@ DEFINE_TAO_TYPE_NAME_HELPER(bool, "i1");
 DEFINE_TAO_TYPE_NAME_HELPER(uint8_t, "ui8");
 DEFINE_TAO_TYPE_NAME_HELPER(int8_t, "i8");
 DEFINE_TAO_TYPE_NAME_HELPER(uint16_t, "ui16");
+DEFINE_TAO_TYPE_NAME_HELPER(int16_t, "i16");
+DEFINE_TAO_TYPE_NAME_HELPER(uint32_t, "ui32");
 DEFINE_TAO_TYPE_NAME_HELPER(int32_t, "i32");
+DEFINE_TAO_TYPE_NAME_HELPER(uint64_t, "ui64");
```

Comment:
@wyzero why is `sized_t` named as 'i64', which is of type `long unsigned int`, aka `uint64_t`.

Reply:
It's workaround to align with the definition of alloc/dealloc in the compiler side.